src/main/java/org/jabref/logic/formatter/bibtexfields/ShortenDOIFormatter.java

```diff
@@ -14,6 +16,7 @@
 public class ShortenDOIFormatter extends Formatter {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ShortenDOIFormatter.class);
+    private static final Predicate<String> SHORT_DOI_FORMAT = Pattern.compile("10/[a-zA-Z0-9]+").asPredicate();
```

Comment:
Please match the start and end (`^`, `$`)of the String as well. I don't believe there are any circumstances we want any surrounding characters